CalcualtorLogic.js

```diff
@@ -98,9 +96,9 @@ class Calculator {
         return div
     }
     onButtonNumberClick(event) {
-        console.log("foo");
+        const buttonElement_ = document.createElement("button");
```

Comment:
this is wrong.

this method meant to be used as event listener
Look at your previous code.

![image](https://user-images.githubusercontent.com/16171515/109430556-f047ff80-79f9-11eb-83f4-5b3b084d34b4.png)

this method will be similar to the function which I draw a circle around it

Reply:
I said in the `onButtonNumberClick` method 
add `onButtonNumberClick` as event listener to the buttons

Reply:
thanks
also you added :
so in getButtonsRow method when you create a button element

Reply:
> thanks
> also you added :
> so in getButtonsRow method when you create a button element

Yes, but you're creating a button in `onButtonNumberClick` method not in `getButtonsRow`
in the `getButtonsRow ` method you are creating a div and few buttons 
afrer creating those buttons (which you're already did) you need to listen to click event on those buttons 

when you listen to an event on an element you need to pass a handler function 
so pass `onButtonNumberClick` has handler

Reply:
sorry just I noticed that you are asking me to create the button element in getButtonRow
just I saw this comment